src/helm/benchmark/metrics/decodingtrust_stereotype_bias_metrics.py

```diff
@@ -0,0 +1,236 @@
+from typing import List, Optional
+from helm.common.request import Sequence
+from helm.common.request import RequestResult
+from helm.benchmark.adaptation.request_state import RequestState
+from .metric import Metric
+from .metric_name import MetricName
+from .statistic import Stat
+
+import numpy as np
+
+
+class StereotypeMetric(Metric):
+    def evaluate_instances(self, request_states: List[RequestState]) -> List[Stat]:
+        stereotype_bias = Stat(MetricName("decodingtrust_stereotype_bias"))
+        rejection_rate = Stat(MetricName("decodingtrust_stereotype_bias_rejection_rate"))
+
+        agreeability_matrix = np.zeros((24, 16, 3))
+        disagreeability_matrix = np.zeros((24, 16, 3))
+        noresponse_matrix = np.zeros((24, 16, 3))
+
+        agree_phrases = [
```

Comment:
Are these phrases tuned for GPT-3 / GPT-4 specifically? Do they also work for other models?

Reply:
Thanks for raising this issue. We manually checked a subset of the responses, and it worked the majority of the time. Although it is not possible to 100% accurately classify every generation, we do not think the misclassification rates on open-source models are extremely high compared to those of GPT-3 / GPT-4.